rio/src/RIO/Process.hs

```diff
@@ -582,11 +606,17 @@ findExecutable name = do
             let loop [] = return $ Left $ ExecutableNotFound name (pcPath pc)
                 loop (dir:dirs) = do
                     let fp0 = dir FP.</> name
-                        fps0 = map (fp0 ++) (pcExeExtensions pc)
+                               -- On Windows, include the name if it has an
+                               -- existing extension
+                        fps0 = (if isWindows && FP.hasExtension fp0
```

Comment:
Any way to share this logic and the `isExecutable` logic from above?

Reply:
There is a way. I will include it in a revised pull request.